hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/scale/ILoadTestS3ABulkDeleteThrottling.java

```diff
@@ -30,6 +30,8 @@
 import java.util.concurrent.ExecutorService;
 
 import com.amazonaws.services.s3.model.DeleteObjectsRequest;
+
+import org.apache.hadoop.fs.store.audit.AuditSpan;
```

Comment:
this should go into the main o.a.h imports. it getting in here because the Preconditions class is where guava is. cherrypicking is a lot easier if imports avoid this area